Backend/dev/test-tool/load-test-service/collection_runner.py

```
@@ -0,0 +1,388 @@
+        candidates = sorted(env_dir.glob('*.json')) if env_dir.exists() else []
+
+    for f in candidates:
+        try:
+            data = json.loads(f.read_text())
+            return {
+                v['key']: _expand(str(v.get('value', '')))
+                for v in data.get('values', [])
+                if v.get('enabled', True) and v.get('key')
+            }
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)